modules/ROOT/pages/_partials/sgw-svr-compatibility.adoc

```diff
@@ -226,6 +238,40 @@ Compatibility Matrix
 | image:ROOT:yes.png[]
 | image:ROOT:yes.png[]
 | image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+
+| 4.0.0
+| 
+| image:ROOT:no.png[]
+| image:ROOT:no.png[]
+| image:ROOT:no.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+
+| 4.0.0
+| Using Scopes and Collections
+| image:ROOT:no.png[]
+| image:ROOT:no.png[]
+| image:ROOT:no.png[]
+| image:ROOT:no.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
+| image:ROOT:yes.png[]
```

Comment:
Do not need to separate scopes and collections here, at this point since 3.1 all supported platforms support scopes and collections